src/collections/_documentation/platforms/python/troubleshooting.md

```diff
@@ -1,19 +1,58 @@
 ---
-title: Contextvars vs thread locals
+title: Troubleshooting
 sidebar_order: 11
 ---
 
-The Python SDK generally does its best to figure out how contextual data such
-as tags set with `sentry_sdk.set_tags` is supposed to flow along your control
-flow. In most usecases this is achieved perfectly fine using [thread
-locals](https://docs.python.org/3/library/threading.html#thread-local-data),
-but there are quite a few situations where this approach fails.
+This section contains various gotchas that are inherent to the Python SDK's
```

Comment:
Perhaps " Troubleshooting helps those using the Python SDK with issues inherent to the design; these issues do not necessarily affect most users. Use this section to help answer these questions:"

Reply:
I've rewritten to be short and direct:

```
We expect most users of the Python SDK not to run into any of the problems documented here.

Use the information in this page to help answer these questions:
```